install.sh

```diff
@@ -70,5 +70,13 @@ else
 
 fi
 
+# Install Haskell
+export BOOTSTRAP_HASKELL_NONINTERACTIVE=1
+curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
+cabal install --lib HUnit
+cabal install --lib gloss
+cabal install hlint
+cabal install hindent
+
 # install Haskell Syntax Highlighting extension
 code --install-extension justusadam.language-haskell
```

Comment:
```suggestion
code --install-extension haskell.haskell
```

I think this one is the best one now :muscle: 

https://marketplace.visualstudio.com/items?itemName=haskell.haskell